tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -132,8 +132,7 @@ public function tearDown() {
    * @throws \CRM_Core_Exception
    */
   public function testGetContribution() {
-    $contributionSettings = $this->enableTaxAndInvoicing();
-    $invoice_prefix = CRM_Contribute_BAO_Contribution::checkContributeSettings('invoice_prefix', TRUE);
```

Comment:
If this incorrectly returned empty then we would be incorrectly checking it was empty - better to test the known value that we set in enableTaxAndInvoicing